CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```

Comment:
Make sure your IDE is configured to auto-format files on save. Your linter should have added line breaks before and after this line.

Reply:
Definitely it was my fault, sorry!